src/app/components/autocomplete/autocomplete.component.ts

```diff
@@ -144,37 +144,37 @@ export class AutocompleteComponent implements OnInit {
     return r;
   }
 
-  myFormatUIFn = (v: any): string => v.name;
+  public myFormatUIFn = (v: any): string => v.name;
 
-  compareByValue(f1: any, f2: any) {
+  public compareByValue(f1: any, f2: any) {
     return f1 && f2 && f1.text === f2.text;
```

Comment:
This autocomplete demo needs to be updated.... as this is the old version that's not supported any more..

Reply:
This will be addressed in the previous PR. Basically all our old demos are not going to be used moving into this site. I want to start with a clean slate since we need to have concise, clear, shareable demos

Reply:
Right. Autocomplete and Card were added a month ago to set up lazy loading. Once we have our own, new components, these will be removed.

Reply:
We should at least remove all the noise inside the files if that is the path we want to take. I don't think it's useful to add a bunch of stuff that we will just need to delete later